README.rst

```diff
@@ -16,11 +16,11 @@
 ====================
  dataflake.fakeldap
 ====================
-This package offers a mock ``pyldap`` library that can be used 
-for testing code relying on ``pyldap`` without having to configure 
+This package offers a mock ``pyldap`` library that can be used
+for testing code relying on ``pyldap`` without having to configure
```

Comment:
one more annoying nitpick, no reason to wrap text here (i.e. 80 chars or w/e), so this should be one line.